.travis.yml

```diff
@@ -12,4 +12,5 @@ cache:
     - $HOME/.npm
 
 script:
+  - npm run build
```

Comment:
Teraz sa to bude zbytocne buildovat dva krat, jedno teraz a potom znovu pre cypress testy. Ale viem s tym zit :D

Reply:
Build teraz netrva dlhsie (stale je to cca 3 minuty), takze je dost mozne ze cypress ked najde zbuildovane sources tak ich prepouzije 🤷‍♂️